DEPLOYMENTS.md

```
@@ -28,6 +28,14 @@ This document tracks all deployed contracts across different networks.
 | CBR7Z27SC7K3AJ7HZEZTFEQ7QJRNQNFF4FI3RBWIEZFTT3E7RHEYQPFL | testnet | 2025-09-28 | done | Purchase Review Contract |
 
 
+## Transaction NFT Contract
+
+### Testnet Deployments
+| Contract ID | Network | Deployed Date | Status | Notes |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)